service/k8sclient/k8sclient.go

```diff
@@ -1195,3 +1196,47 @@ func (c *K8sClient) checkOperatorStatus(installedVersions []string, expectedAPIV
 	}
 	return OperatorStatusNotInstalled
 }
+
+// GetClusterPods returns list of cluster's pods. It finds them by given cluster
+// name.
+func (c *K8sClient) GetClusterPods(ctx context.Context, clusterName string) (*common.PodList, error) {
+	list := &common.PodList{}
```

Comment:
looks like it shouldn't be pointer here or in line 1208 we shouldn't use &

Reply:
Fixed